PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -539,6 +538,13 @@
     )
 )
 
+# Deprecation/backcomp
+for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_102Xv1, run2_nanoAOD_106Xv1:
+  # post 106X
+  modifier.toModify(subJetTable.variables,
```

Comment:
```suggestion
  # post 106X
_modifier  = (run2_miniAOD_80XLegacy | run2_nanoAOD_94X2016 | run2_nanoAOD_94XMiniAODv1 |
    run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_102Xv1 | run2_nanoAOD_106Xv1)
  _modifier.toModify(subJetTable.variables,
```
may be a bit more readable

Reply:
@slava77 let me re-ask the question from above, is there a way to fetch these "by age". For this kind of deprecation it would be very neat to have logic like 
```
for modifier in modifiers_older_then[deprecated_here]:
   do old stuff that we don't do anymore
```

Reply:
> @slava77 let me re-ask the question from above, is there a way to fetch these "by age".

I don't clearly see how this can be encoded at the level of top-level modifier.
One solution could be to refactor these top-level modifiers to smaller pieces, like `nano_rerun_blah` of different kinds and then chain them into a given `run2_nanoAOD_XYZ`

Reply:
ok, cheers